src/container/WeatherContainer/WeatherContainer.tsx

```diff
@@ -55,12 +72,41 @@ export const WeatherContainer: React.FC = () => {
     };
   };
 
+  const CityInput = () => {
+    return (
+      <div className="chooser">
+        <input
+          value={userDefinedCity}
+          onChange={checkRegexExpression}
+          placeholder="Your City"
+        />
+        <Button onClick={onButtonClick}>Send</Button>
+      </div>);
+  };
+
+  const checkRegexExpression = (inputElement: React.ChangeEvent<HTMLInputElement>) => {
+    const value: string = inputElement.target.value;
+
+    if (!value) {
+      setUserDefinedCity('');
+    } else if (userInputRegex.test(value)) {
```

Comment:
what if it will be value but not match with regex?

Reply:
Nothing